app/src/main/res/layout/activity_setup.xml

```diff
@@ -1,10 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
-<fragment xmlns:android="http://schemas.android.com/apk/res/android"
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Why do we need a framelayout?

Reply:
View binding will not generate a reference for `fragment` when it's root layout. So we are just wrapping it in `FrameLayout` to generate that.